sdks/python/setup.py

```diff
@@ -165,6 +165,7 @@ def get_version():
     'requests>=2.24.0,<3.0.0',
     'typing>=3.7.0,<3.8.0; python_full_version < "3.5.3"',
     'typing-extensions>=3.7.0,<3.8.0',
+    'mmh3>=2.5.1,<2.5.2',
```

Comment:
(I thought I already added this comment here but for some reason I don't see it...)
Let's make the upper bound more flexible `mmh3>=2.5.1,<3.0`, or remove the obligatory dependency on mmh3 as we do for snappy.  @aaltay do you have a preference on this?
For the record, Windows tests on the PR are passing. AFAIK, previously we didn't add a dep on mmh3 we observed installation errors on Google internal Windows test.

Reply:
It is better to avoid adding a new dependency if possible especially a one with platform specific differences. If we need to add it, it is better to make it optional similar to snappy.